runtime/tests/port/si.c

```diff
@@ -1893,7 +1919,7 @@ j9sysinfo_test_get_cwd3(struct J9PortLibrary *portLibrary)
 		outputComment(PORTLIB, "CWD = %s\n", buffer);
 	}
 
-	rc = memcmp(utf8, buffer, strlen(buffer));
+	rc = memcmp(cmpbuf, buffer, strlen(buffer));
 	if (0 != rc) {
 		outputErrorMessage(PORTTEST_ERROR_ARGS, "invalid directory rc: %d\n", rc);
```

Comment:
It would help if this message included the actual and expected directories.

Reply:
They are printed by previous statements.